### PR TITLE
add image-picker v0.2.4 with git autoupdate

### PR DESCRIPTION
**Pull request by maruilian11**

### CODE REVIEW

File: ajax/libs/image-picker/package.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    "bower_components",
+    "test",
+    "tests"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/rvera/image-picker.git",
+    "basePath": "image-picker",
+    "files": [
+      "**"
```
Comment (PeterDaveHello) [inline ajax/libs/image-picker/package.json]: Should be `**/*` not `**`.

**Comment by PeterDaveHello** [comment]
Missing minified css file.

**Comment by maruilian11** [comment]
fixed

File: ajax/libs/image-picker/package.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    "bower_components",
+    "test",
+    "tests"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/rvera/image-picker.git",
+    "basePath": "image-picker",
+    "files": [
+      "**/*"
```
Comment (PeterDaveHello) [inline ajax/libs/image-picker/package.json]: Should not use `**/*` in non-dist folder

**Comment by drewfreyling** [comment]
@maruilian11 There are no files changed??

**Comment by PeterDaveHello** [comment]
Messed up.

**Comment by maruilian11** [comment]
@drewfreyling sorry, i make some mistakes, i will send new pr for it.

**Pull request closed without merging**